src/sssom/io.py

```diff
@@ -98,17 +99,24 @@ def parse_file(
     write_table(doc, output, embedded_mode)
 
 
-def validate_file(input_path: str, validation_types: List[SchemaValidationType]) -> None:
+def validate_file(
+    input_path: str,
+    *,
```

Comment:
what happens with implementations that call `validate_file(a,b)` without the argument names, if the `*` is the second option?

Reply:
I guess this is breaking, but it's good form to have arguments with defaults be keyword only. I can revert this

Reply:
fixed in d96e3eb and 8b4011f

Reply:
Thank you!